dvc/repo/experiments/init.py

```diff
@@ -251,7 +248,7 @@ def init(
         metrics_no_cache=compact([context.get("metrics"), live_metrics]),
         plots_no_cache=compact([context.get("plots"), live_plots]),
         force=force,
-        **{"checkpoints" if with_live else "outs": compact([models])},
+        **{"checkpoints" if type == "dl" else "outs": compact([models])},
```

Comment:
I guess this breaks API but cleary `type "dl"` is no longer a good name and could be just `checkpoint`

Reply:
Could we alias the user facing option, without removing old version, so it's backwards compatible?

Reply:
Agreed. I guess I can alias it.

Reply:
Actually, I don't think I can have hidden/aliased choices, so probably the best we can do is leave them both as user-facing.

Reply:
Or we can remove `dl`. It's new, it's not used in docs outside of the cmd ref, and it's not a command that's likely to be automated.